site/src/api/typesGenerated.ts

```diff
@@ -2559,6 +2559,7 @@ export interface WorkspaceApp {
     readonly healthcheck: Healthcheck;
     readonly health: WorkspaceAppHealth;
     readonly hidden: boolean;
+    readonly open_in: string;
```

Comment:
Defining this as a typed const in the Go code will mean this becomes a proper type in the generated TypeScript code.